wrappers/python/aries_askar/store.py

```diff
@@ -626,4 +626,11 @@ async def __aenter__(self) -> Session:
     async def __aexit__(self, exc_type, exc, tb):
         session = self._session
         self._session = None
+
+        if session.is_transaction and session.handle:
+            if exc:
+                await session.rollback()
+            else:
+                await session.commit()
+
```

Comment:
I think the `close()` should be in an else block, as commit and rollback both close the session.